superset-frontend/src/components/Checkbox/Checkbox.stories.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { useArgs } from '@storybook/client-api';
+import Checkbox from './';
+
+export default {
+  title: 'Checkbox',
+  component: Checkbox,
+};
+
+// eslint-disable-next-line no-unused-vars
+export const Component = _args => {
+  const [{ checked, style }, updateArgs] = useArgs();
```

Comment:
Pretty cool, I didn't know this even existed, actually. Nice that it lets the components do the toggling.

I still like the idea of knobs, too, so you can control any given component from a common location. I wonder if there's a way that both can co-exist, e.g. this `updateArgs` toggles the `checked` prop _and_ toggles the knob's checkbox.

Reply:
FWICT storybook seems to be trying to move people to args instead of knobs, since args is part of original package instead of an addon: https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#knobs-to-custom-args
I put together a draft of the buttons as an arg/controls to see if we could migrate knobs to args, and it seems to work, but wdyt?

Here's a view of the checkbox and button as args/controls (ignore the knob stories for now): 
![Screen Recording 2020-09-08 at 6 12 57 PM](https://user-images.githubusercontent.com/5186919/92543141-7d568c80-f1ff-11ea-8807-ae8c85f577a9.gif)

Reply:
Args/controls LGTM!